include/ruby/internal/intern/vm.h

```diff
@@ -95,6 +95,7 @@ VALUE rb_check_funcall(VALUE recv, ID mid, int argc, const VALUE *argv);
  */
 VALUE rb_check_funcall_kw(VALUE recv, ID mid, int argc, const VALUE *argv, int kw_splat);
 
+#ifdef RBIMPL_ATTR_DEPRECATED_INTERNAL
```

Comment:
`RBIMPL_ATTR_DEPRECATED_INTERNAL` is not defined here because `include/ruby/backward.h` is not included at this point yet: https://github.com/ruby/tk/issues/75

https://github.com/ruby/ruby/pull/15435 should fix this.